tests/stats/test_stats.py

```diff
@@ -0,0 +1,55 @@
+from utils import interfaces, weblog, features, scenarios, bug
+from utils.tools import logger
+
+"""
+Test scenarios we want:
+    * Generate N spans that will be aggregated together
+        - Must aggregate by:
+            - HTTP status code
+            - peer service tags (todo: can we just rely on the defaults?)
```

Comment:
I only know about 40% of this sentence.

I presume 1 peer service will be coming from the System-tests http client ? 
Do you plan to incoroporate a buddy service (IIRC we have examples of it in system-tests , a second service used as another peer)

Reply:
So for the purposes of testing client stats we don't even need a "buddy service", we can just manually add a peer.service tag to verify that the aggregation happens (I'm also adding a test for this in the parametric tests which will be a separate PR)